[GridPlus] Update modules to unlock future Lattice functionality

alex-miller-0 opened the pull request:
GridPlus has updated the EVM signing pathway in Lattice firmware,
which has not yet been released. Additionally, requesters can now
include ABI definitions with signing requests, which are used by
Lattice firmware to decode calldata in place.
All updates are backward compatable.
Updates:
* https://github.com/GridPlus/gridplus-sdk/compare/v1.1.6...v1.2.4
* https://github.com/GridPlus/eth-lattice-keyring/compare/v0.6.1...v0.7.3

digiwand commented:
Comparing:
https://github.com/GridPlus/eth-lattice-keyring/compare/v0.6.1...v0.7.1
&
https://github.com/GridPlus/gridplus-sdk/compare/v1.1.6...v1.2.0

alex-miller-0 committed:
[GridPlus] Updates Lattice-related modules to unlock functionality
GridPlus has updated the EVM signing pathway in Lattice firmware,
which has not yet been released. Additionally, requesters can now
include ABI definitions with signing requests, which are used by
Lattice firmware to decode calldata in place.
All updates are backward compatable.
Updates:
* https://github.com/GridPlus/gridplus-sdk/compare/v1.1.6...v1.2.4
* https://github.com/GridPlus/eth-lattice-keyring/compare/v0.6.1...v0.7.3
